docs/en/engines/table-engines/integrations/embedded-rocksdb.md

```diff
@@ -15,11 +15,13 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
     name1 [type1] [DEFAULT|MATERIALIZED|ALIAS expr1],
     name2 [type2] [DEFAULT|MATERIALIZED|ALIAS expr2],
     ...
-) ENGINE = EmbeddedRocksDB PRIMARY KEY(primary_key_name)
+) ENGINE = EmbeddedRocksDB([ttl, read_only]) PRIMARY KEY(primary_key_name)
 ```
 
-Required parameters:
+Engine parameters:
 
+- `ttl` - time to live. TTL is accepted in seconds. Not specifying/passing or non-positive TTL behaves like TTL = infinity.
```

Comment:
```suggestion
- `ttl` - time to live for values. TTL is accepted in seconds. If TTL is 0 or negative behaves, regular RocksDB instance is used (without TTL).
```

Reply:
`ttl` is `Int32`, so a negative number is also acceptable here

Reply:
But we don't care about negative numbers only 0 or positive